recipes/libavrocpp/1.10.1/conanfile.py

```diff
@@ -0,0 +1,100 @@
+import os
+
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+from conans.tools import Version
+
+
+class LibavrocppConan(ConanFile):
+    name = "libavrocpp"
+    version = "1.10.1"
+    license = "Apache License 2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Avro is a data serialization system."
+    homepage = "https://avro.apache.org/"
+    topics = ("serialization", "deserialization")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_snappy": [True, False],
+    }
+    default_options = {"shared": False, "fPIC": True, "with_snappy": True}
+    generators = "cmake", "cmake_find_package"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+        compiler = str(self.settings.compiler)
+        compiler_version = Version(self.settings.compiler.version.value)
```

Comment:
The value is not required 
```suggestion
        compiler_version = Version(self.settings.compiler.version)
```